kuka_eki_hw_interface/package.xml

```diff
@@ -20,4 +20,5 @@
   <depend>hardware_interface</depend>
   <depend>joint_limits_interface</depend>
   <depend>roscpp</depend>
+  <depend>angles</depend>
```

Comment:
Sorting here again.

And `angles` is a header only library, so no need to make it a `depend`.